### PR TITLE
GitHub Actions

### PR DESCRIPTION
**Pull request by Dahlgren**

### CODE REVIEW

**Comment by Dahlgren** [comment]
https://github.com/Dahlgren/ArmaScriptProfiler/actions/runs/107608561

**Comment by dedmen** [comment]
Ready for merge or more wip?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        uses: actions/download-artifact@v1
+        with:
+          name: ArmaScriptProfiler.so
+          path: '@ArmaScriptProfiler'
+
+      - name: Download Windows 64 bit
+        uses: actions/download-artifact@v1
+        with:
+          name: ArmaScriptProfiler_x64.dll
+          path: '@ArmaScriptProfiler'
```
Comment (Dahlgren) [inline .github/workflows/ci.yml:87]: intercept folder

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        uses: actions/download-artifact@v1
+        with:
+          name: 'mod'
+          path: '@ArmaScriptProfiler'
+
+      - name: Download Linux 32 bit
+        uses: actions/download-artifact@v1
+        with:
+          name: ArmaScriptProfiler.so
+          path: '@ArmaScriptProfiler'
```
Comment (Dahlgren) [inline .github/workflows/ci.yml:81]: intercept folder

**Comment by Dahlgren** [comment]
Should be good to go now, Windows works as intended with `-intignorecert`, Linux needs the other fix as well to work with the server

**Pull request merged**